cards/src/main/java/com/handonn/finapp/cards/service/ICardService.java

```diff
@@ -0,0 +1,21 @@
+package com.handonn.finapp.cards.service;
+
+
+import com.handonn.finapp.cards.model.CardDto;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+
+public interface ICardService {
+
+    void create(CardDto cardDto);
+
+    List<CardDto> getAllCard(Pageable pageable);
```

Comment:
Should return Page<CardDto>

Reply:
Add filter for getAllCard, to filter by cardType and totalAmount in range (min, max)

Page<CardDto> getAllCard(Pageable pageable, CardCriteria criteria);
With
class CardCriteria {
  private CardType cardType;
  private BigInteger minTotalAmount;
  private BigInteger maxTotalAmount;
}